### DISCUSSION TITLE
2023/05/nuclear-pollution/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#80] [General]
# 2023/05/nuclear-pollution/

https://xiangyun.rbind.io/2023/05/nuclear-pollution/

**Comment by XiangyunHuang** [comment]
> **提示**
>
>如果 **spaMM** 包的函数 `fitme()` 来拟合空间线性混合效应模型，它可以替代 **nlme** 包的 `gls()` 函数。二者拟合结果完全一致，值得注意，由于 **spaMM** 包自己有一套符号表述，按照本文中的模型和数学符号约定，输出结果需要对应一下。

```r
library(spaMM)
fit_rongelap_gauss <- fitme(log(counts / time) ~ 1 + Matern(1 | cX + cY),
  data = rongelap, fixed = list(nu = 0.5), method = "REML"
)
summary(fit_rongelap_gauss)
```
```
formula: log(counts/time) ~ 1 + Matern(1 | cX + cY)
REML: Estimation of corrPars, lambda and phi by REML.
      Estimation of fixed effects by ML.
Estimation of lambda and phi by 'outer' REML, maximizing restricted logL.
family: gaussian( link = identity ) 
 ------------ Fixed effects (beta) ------------
            Estimate Cond. SE t-value
(Intercept)    1.813   0.1088   16.66
 --------------- Random effects ---------------
Family: gaussian( link = identity ) 
                   --- Correlation parameters:
       1.nu       1.rho 
0.500000000 0.005891123 
           --- Variance parameters ('lambda'):
lambda = var(u) for u ~ Gaussian; 
   cX + cY  :  0.2934  
# of obs: 157; # of groups: cX + cY, 157 
 -------------- Residual variance  ------------
phi estimate was 0.035991 
 ------------- Likelihood values  -------------
                        logLik
logL       (p_v(h)): -86.92330
Re.logL  (p_b,v(h)): -88.22257
```

输出结果中，Re.logL 表示限制对数极大似然，残差的方差 phi 表示文中的块金效应 $\tau^2 = 0.035991$ ，空间随机效应的方差 lambda 表示文中的 $\sigma^2 = 0.2934$，空间相关性参数 rho 的倒数表示文中的范围参数 $\phi = 1/0.005891123 = 169.7469$

**Reply by XiangyunHuang** [reply]
在空间线性混合效应模型中，值得注意，此时，块金效应 $\tau^2$ 和模型残差 $\sigma_{\epsilon}^2$ 是合并在一起的。

**Comment by XiangyunHuang** [comment]
克里斯托弗·诺兰的新电影《奥本海默》上映了，奥本海默被称为原子弹之父，与之合作的泰勒被称为氢弹之父，二战后，前苏联、美国、英国、法国、中国，陆续紧锣密鼓地、争先恐后地研制核武器。科学家都是爱国的，却都不幸沦为政治家、野心家的工具，奥本海默、爱因斯坦、波尔等一批从事理论和实验的著名物理科学家感到深深的内疚、有罪。
1954 年美国在太平洋比基尼环礁上的那次氢弹核试验是人类历史上规模最大的一次实验，这颗氢弹的当量相当于广岛原子弹的 1000 倍。由于实际当量超出预估的 2.5 倍，气象条件也不利，产生大面积核辐射，一些基地、居民都没来得及撤离，几百公里外的朗格拉普环礁也受到核辐射，半个世纪后，岛上的核辐射强度仍然很高，不适合人类居住。
